plugins/bgp/model/bgp.proto

```diff
@@ -14,12 +14,13 @@ message BgpConf {
 
 // global configuration
 message GlobalConf {
-    uint32 as = 1;
-    string router_id = 2;
-    int32 listen_port = 3;
-    repeated string listen_addresses = 4;
-    repeated uint32 families = 5;
-    bool use_multiple_paths = 6;
+    string name = 1; //may not be needed, use for the handler
+    uint32 as = 2;
```

Comment:
We do not need name with global conf; but we do need a name with the peer (we need to identify the peer - there are multiple peers; we do not need to identify global conf - it's a singleton)